### ISSUE TITLE
Unable to schedule charge (anymore)

### ISSUE THREAD

**Issue by mikey4321** [ngardiner/TWCManager#368]
It used to work, but for the past few versions I've not been able to get scheduled charging to work as expected.

Current settings:

1. Car has scheduled charging and scheduled departure OFF (i.e. it just charges if available).  Excess solar charging works fine with stop/start/adjust amount.

2. TWC settings as follows:  
a) fresh install v1.2.4, included deleting and recreating /etc/twcmanager/settings.json and config.json with latest

b) Scheduled charging set to 2300-0700.  No difference if I do this as a single setting, or for each day.

c) Charge limit is at the single value of 70%. (I used to use the 4 different limits successfully in old versions, but this doesn't seem to work any more either).

Basically the car doesn't charge overnight to top up to the 70% value as it used to.  Any suggestions or key parts of the logs that could help narrow down the issue?

Mike
